asset/scss-about/_section-about.scss

```diff
@@ -0,0 +1,59 @@
+.main_program {
```

Comment:
- [ ] The main program section's content should be aligned horizontally on the mobile screen. It appears that it breaks at lower points. Kindly work on that!

>A screenshot from your project.
![Screenshot6](https://user-images.githubusercontent.com/47625407/124309153-95badd80-db6a-11eb-8e17-cc6fa890c700.png)

_**Please refer to the review summary and the given design link for more information.**_